src/commands/start.yml

```diff
@@ -9,16 +9,14 @@ description: |
   Note: This command needs to be started with a VM executor (Docker-based executors are not yet supported).
 
 parameters:
-  install-awslocal:
+  install_awslocal:
```

Comment:
If the linter tries to enforce this, it seems like this would be a best practice.
However, my only real concern is: Is this change backwards compatible.
- If yes: Nice.
- If not: Then we have two options.
  - We don't go with this change (and for example adjust the linter to not complain about it)
  - We do a major bump instead of a minor one.

For me all these options are okay to be honest. The orb does not have a huge user-base (the impact isn't too big), the naming seems to be a convention / best practice (we should adjust it at some point anyways), and this PR adds a lot of new features (which also justifies a major release).

Reply:
There is an exclude option that can turn off certain rule checks.
I reverted the change in the `start` command to keep backwards compatibility, however since `tools` is a new command I keep the best practice for future-proofing and in the next major version we can unify the format.